app/models/event.rb

```diff
@@ -18,7 +18,7 @@ class Event < ActiveRecord::Base
   attr_accessor :repeat_ends_string
 
   COLLECTION_TIME_FUTURE = 10.days
-  COLLECTION_TIME_PAST = 15.minutes
+  COLLECTION_TIME_PAST = 300.minutes
```

Comment:
if this is the fix that's awesome - I guess what this means is that any event that was started in the last 5 hours will stay on the upcoming events page.    Ah, but you're handling that with the `remove_past_events` method - I see